app/views/layouts/application.html.haml

```diff
@@ -1,6 +1,13 @@
 !!!
 %html
   %head
+    - if test?
+      :javascript
+        window.errors = []
+        window.onerror = function(a, b, c){
+          window.errors.push([a, b, c]);
+          return false; // false: don't swallow exceptions.
+        }
```

Comment:
This backs the implementation of `assert_js_ok`. `assert_js_ok` must be run after every JS test. See kuina spec_helper and also `def assert_js_ok`